src/include/stir/recon_buildblock/ChainedBinNormalisation.h

```diff
@@ -103,15 +103,43 @@ ChainedBinNormalisation(shared_ptr<BinNormalisation> const& apply_first,
   */
   virtual void apply(RelatedViewgrams<float>& viewgrams,const double start_time, const double end_time) const;
 
+  virtual void apply(ProjData&,const double start_time, const double end_time) const;
+
+virtual void apply_only_first(RelatedViewgrams<float>& viewgrams,const double start_time, const double end_time) const;
```

Comment:
I'm a tiny bit reluctant to have these. why do we need them?

Reply:
Are need in the estimation. We have to apply only the normalisation L:1040